setup.py

```diff
@@ -107,6 +107,7 @@ def get_build_env():
             **os.environ.copy(),
             "TT_METAL_HOME": Path(__file__).parent,
             "TT_METAL_ENV": "production",
+            "CXX": "clang++-17",
```

Comment:
This passes it in ? Doesn't need as a `-D` arg?

Reply:
yep